css/style.css

```diff
@@ -287,72 +287,356 @@
 
 /* Описание стилей Третьей и Четвертой секции */
 
+/* Описание третьей секции */
+
 .wrap-information {
-  position: relative;
+  width: 1440px;
+  height: 844px;
+  padding-bottom: 9px;
+  background: url(../img/jpg/2013-tesla-model-s-cockpit.jpg) 0 0/contain no-repeat;
 }
 
+/* .wrap__pic {
+  width: inherit;
+  height: auto;
+  background: url(../img/jpg/2013-tesla-model-s-cockpit.jpg) -19px 37px/cover no-repeat;
+  } */
+
 .info__title {
-  position: relative;
-  z-index: 2;
   color: #e51937;
   font-size: 71px;
   font-weight: 600;
   transform: translate(77px, 20px);
 }
 
 .info__img {
-  width: 1273px;
+  max-width: 1273px;
   height: 843px;
   position: absolute;
   transform: translate(0, -128px);
 }
 
 .red-first {
-  position: absolute;
-  transform: translate(226%, -185px);
   background-color: #e51937;
   color: #fff;
-  max-width: 441px;
-  height: 900px;
+  max-width: 439px;
+  height: 907px;
   box-sizing: border-box;
-  padding: 54px 0 0 55px;
+  padding: 58px 47px 0 52px;
+  float: right;  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)